tdrs-backend/tdpservice/settings/common.py

```diff
@@ -397,3 +398,27 @@ class Common(Configuration):
         'AMS_CLIENT_SECRET',
         ''
     )
+
+    # ------- SFTP CONFIG
+    ACFTITAN_SERVER_ADDRESS = os.getenv('ACFTITAN_HOST', '')
+    """
+    To be able to fit the PRIVATE KEY in one line as environment variable, we replace the EOL 
+    with an underscore char.
+    The next line replaces the _ with EOL before using the PRIVATE KEY
+    """
+    ACFTITAN_LOCAL_KEY = os.getenv('ACFTITAN_KEY', '').replace('_', '\n')
+    ACFTITAN_USERNAME = os.getenv('ACFTITAN_USERNAME', '')
+    ACFTITAN_DIECTORY = os.getenv('ACFTITAN_DIECTORY', '')
+
+    # -------- CELERY CONFIG
+    REDIS_URI = os.getenv(
+        'REDIS_URI',
+        'redis://redis-server:6379'
```

Comment:
Will this be true in both local and deployed contexts? If I recall, we had this set to just localhost for one iteration -- I think using docker.

Reply:
In local, we skip adding the REDIS_URI, so this will set it to docker instance name, however for deployement we add REDIS_URI = redis://localhost:6379.